src/libData/AccountData/AccountStoreBase.tpp

```diff
@@ -127,14 +127,15 @@ bool AccountStoreBase<MAP>::UpdateAccounts(const Transaction& transaction,
 
   Account* fromAccount = this->GetAccount(fromAddr);
   if (fromAccount == nullptr) {
+    LOG_GENERAL(WARNING, "sender " << fromAddr.hex() << " not exist");
     // FIXME: remove this, temporary way to test transactions, should return
     // false
-    LOG_GENERAL(WARNING,
-                "AddAccount... FIXME: remove this, temporary way to "
-                "test transactions, should return false in the future");
-    this->AddAccount(fromAddr, {10000000000, 0});
-    fromAccount = this->GetAccount(fromAddr);
-    // return false;
+    // LOG_GENERAL(WARNING,
```

Comment:
Can we just remove the commented out code entirely?

Reply:
sure